### PR TITLE
AD query V2-fix

### PR DESCRIPTION
**Pull request by AradCarmi**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://github.com/demisto/etc/issues/48090)

## Description
An enhancement for enable/disable command adding the restore_user argument.
  
## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Active_Directory_Query/Integrations/Active_Directory_Query/Active_Directory_Query.yml [hunk 1]
```diff
@@ -251,6 +251,8 @@ script:
     - name: base-dn
       description: Root. For example, DC=domain,DC=com). By default, the Base DN configured
         for the instance is used.
+    - name: restore_user
+      description: "If true the command will enable the user with his restored options"
```
Comment (ShahafBenYakir) [inline Packs/Active_Directory_Query/Integrations/Active_Directory_Query/Active_Directory_Query.yml:255]: Should have predefined values and default value

**Review by Shellyber — COMMENTED** [review]
Let's add type hints and docstrings  to the commands and a test

File: Packs/Active_Directory_Query/ReleaseNotes/1_4_10.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Active Directory Query v2
+- Added support for the *restore_user* argument in the ***enable/disable*** user command.
```
Comment (Shellyber) [inline Packs/Active_Directory_Query/ReleaseNotes/1_4_10.md:4]:
```suggestion
- Added new argument ***restore_user*** to **ad-enable-account** and **ad-disable-account** commands
```

**Review by Shellyber — COMMENTED** [review]
Nice!

File: Packs/Active_Directory_Query/ReleaseNotes/1_4_10.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Active Directory Query v2
+- Added new argument ***restore_user*** to **ad-enable-account** and **ad-disable-account** commands.
```
Comment (Shellyber) [inline Packs/Active_Directory_Query/ReleaseNotes/1_4_10.md:4]:
```suggestion
- Added support for ***restore_user*** to **ad-enable-account** and **ad-disable-account** commands.
```

**Comment by ShahafBenYakir** [comment]
@AradCarmi need to add the new argument to the integration doc

File: Packs/Active_Directory_Query/Integrations/Active_Directory_Query/README.md [hunk 4]
```diff
@@ -418,10 +418,11 @@ Requires `Read userAccountControl` and `write userAccountControl` permissions.
 
 ##### Input
 
-| **Argument Name** | **Description** | **Required** |
-| --- | --- | --- |
-| username | The username of the account to enable (sAMAccountName). | Required |
-| base-dn | Root (e.g., DC=domain,DC=com). By default, the Base DN configured for the instance will be used. | Optional |
+| **Argument Name** | **Description**                                                                                  | **Required** |
+| --- |--------------------------------------------------------------------------------------------------|-------------|
```
Comment (ShahafBenYakir) [inline Packs/Active_Directory_Query/Integrations/Active_Directory_Query/README.md:422]:
```suggestion
| --- | --- | --- |
```

**Review by ShahafBenYakir — APPROVED** [review]

**Comment by AradCarmi** [comment]
This PR forced due to issue in the build process ([link](https://code.pan.run/xsoar/content/-/pipelines/2902583))

**Pull request merged**